File: Aprendendo-C++/Exemplos/entradaDados.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>

using namespace std;

int main(){

    double salario1, salario2;
    string nome1, nome2;
    int idade;
    char sexo;

    cout << "Nome da primeira pessoa: " << endl;
    getline(cin,nome1);

    cout<< "Salario da primeria pessoa: " << endl;
    cin >> salario1;

    cout<< "Nome da segunda pessoa: " << endl;
    cin.ignore(INT_MAX, '\n');
    getline(cin, nome2);

    cout<< "Salario da segunda pessoa "<< endl;
    cin >> salario2;

    cout<< "Digite uma idade: ";
    cin >> idade;

    cout<<"Digite um sexo (F/M): ";
    cin >> sexo;

    cout << fixed << setprecision(2);
    cout << "Nome 1: " << nome1 << endl;
    cout << "Salario: " << salario1 << endl;
    cout << "Nome 2: " << nome2 << endl;
    cout << "Salario: " << salario2 << endl;
    cout << "Idade: " << idade << endl;
    cout << "Sexo: " << sexo << endl;

return 0;
}

File: Aprendendo-Java/Exemplos/Exemplos/src/exemplos/entradaDados.java
package exemplos;
import java.util.Locale;
import java.util.Scanner;

public class entradaDados {
    public static void main(String[] args){
    
    Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);
    
    String nome1, nome2;
    double salario1, salario2;
    int idade;
    char sexo;
    
        System.out.print("Nome da primeira pessoa: ");
        nome1 = sc.nextLine();

        System.out.print("Salario da primeira pessoa: ");
        salario1 = sc.nextDouble();

        System.out.print("Nome da segunda pessoa: ");
        sc.nextLine();
        nome2 = sc.nextLine();

        System.out.println("Salario da segunda pessoa: ");
        salario2 = sc.nextDouble();
    
        System.out.println("Digite uma idade: ");
        idade = sc.nextInt();
        
        System.out.println("Digite uma sexo de (F/M): ");
        sexo = sc.next().charAt(0);
        
        System.out.println("Nome 1:" + nome1);
        System.out.println("Salario 1: "+ String.format("%.2f", salario1));
        
        
        System.out.println("Nome 2:" + nome2);
        System.out.println("Salario 2: "+ String.format("%.2f", salario2));
        
        System.out.println("idade: "+ idade);
        System.out.println("sexo: "+ sexo);
        
        sc.close();
    } 
}
